fw/Core/Hitcon/App/TamaApp.cc

```diff
@@ -253,6 +277,36 @@ void TamaApp::UpdateFrameBuffer() {
       break;
   }
 }
+
+void TamaApp::XbOnButton(button_t button) {
+  // TODO: Handle all XBoard button here
+}
+void TamaApp::XbUpdateFrameBuffer() {
+  // TODO: Handle all XBoard frame here
+}
+
+void TamaApp::OnXBoardRecv(void* arg) {
+  PacketCallbackArg* packet = reinterpret_cast<PacketCallbackArg*>(arg);
+  switch ((TAMA_XBOARD_PACKET_TYPE)packet->data[0]) {
+    // TODO: Handle XB game logic here
+    case TAMA_XBOARD_PACKET_TYPE::PACKET_CONFIRM:
+      // TODO: Confirm battle
+      break;
+    case TAMA_XBOARD_PACKET_TYPE::PACKET_SCORE:
+      // TODO: Update score
+      break;
+    case TAMA_XBOARD_PACKET_TYPE::PACKET_END:
+      // TODO: End game
+      break;
+    case TAMA_XBOARD_PACKET_TYPE::PACKET_LEAVE:
+      badge_controller.BackToMenu(this);
+      return;  // Exit immediately
+    default:
+      my_assert("WTF");
```

Comment:
This is a no op because "WTF" is truthful (non zero address), is this intentional?